learnbase/src/main/java/com/google/sps/servlets/RecommendationsServlet.java

```diff
@@ -128,4 +128,4 @@ public double getCosineDistance(HashMap<String, ArrayList<Double>> words2vecs, S
     return distance;
   }
 
-}
\ No newline at end of file
+}
```

Comment:
Refactor to smaller functions (<= 15-20 LOC), add a newline to the end of the file (stupid style thing)